src/translations/network-modifications-fr.json

```diff
@@ -38,5 +38,6 @@
     "network_modifications/tabular/GENERATOR_MODIFICATION": "modifications de générateurs",
     "network_modifications/tabular/LOAD_MODIFICATION": "modifications de consommations",
     "network_modifications/BY_FORMULA_MODIFICATION": "Modification par formule {computedLabel}",
-    "network_modifications/tabular/VOLTAGE_LEVEL_MODIFICATION": "modifications de postes"
+    "network_modifications/tabular/VOLTAGE_LEVEL_MODIFICATION": "modifications de postes",
+    "network_modifications/tabular/TWO_WINDINGS_TRANSFORMER_MODIFICATION": "modifications de transformateurs à 2 enroulements"
```

Comment:
this is a very long label...
![image](https://github.com/gridsuite/gridstudy-app/assets/123802855/05e4945f-6de3-4e71-9996-e0d6966059d5)

Reply:
It is, what do you suggest ?
For me the "modifications" at first is too much, but it have been decided like that.
Anyway it's a more global problem in all the app.

Reply:
I think we should do one iteration with a PO. I went for this for deletion by filter:
![image](https://github.com/gridsuite/gridstudy-app/assets/123802855/07cf888e-996a-4036-94be-58e6bed49681)
`($equipmentType)`

Reply:
Ok, it's better IMO and we should homogenize anyway